curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md

```diff
@@ -18,13 +18,13 @@ localeTitle: Use uma classe CSS para estilizar um elemento
 ```yml
 tests:
   - text: Seu elemento <code>h2</code> deve estar vermelho.
-    testString: 'assert($("h2").css("color") === "rgb(255, 0, 0)", "Your <code>h2</code> element should be red.");'
+    testString: assert($("h2").css("color") === "rgb(255, 0, 0)");
   - text: Seu elemento <code>h2</code> deve ter a classe <code>red-text</code> .
-    testString: 'assert($("h2").hasClass("red-text"), "Your <code>h2</code> element should have the class <code>red-text</code>.");'
+    testString: assert($("h2").hasClass("red-text"));
   - text: Sua folha de estilo deve declarar uma classe de <code>red-text</code> e ter sua cor definida como vermelha.
-    testString: 'assert(code.match(/\.red-text\s*\{\s*color\s*:\s*red;\s*\}/g), "Your stylesheet should declare a <code>red-text</code> class and have its color set to red.");'
+    testString: assert(code.match(/\.red-text\s*\{\s*color\s*:\s*red;\s*\}/g));
   - text: 'Não use declarações de estilo inline como <code>style=&quot;color: red&quot;</code> em seu elemento <code>h2</code> .'
-    testString: 'assert($("h2").attr("style") === undefined, "Do not use inline style declarations like <code>style="color&#58; red"</code> in your <code>h2</code> element.");'
+    testString: assert($("h2").attr("style") === undefined);
```

Comment:
I suggest the text translated
```suggestion
    testString: 'assert($("h2").attr("style") === undefined, "Não use declarações de estilo embutido como <code>style="color&#58; red"</code> em seu elemento <code>h2</code>.");'
```

Reply:
We don't need to add assert messages.  Instead, the `text` section is the assert message.